SegmentEditorSurfaceCut/SegmentEditorSurfaceCut.py

```diff
@@ -133,7 +149,8 @@ def test_SurfaceCut1(self):
     segStatLogic.showTable(resultsTableNode)
 
     self.delayDisplay("Check a few numerical results")
-    self.assertEqual( round(segStatLogic.statistics["Tumor","LM volume cc"]), 16)
-    self.assertEqual( round(segStatLogic.statistics["Background","LM volume cc"]), 3010)
+
+    stats = segStatLogic.getStatistics()
+    self.assertEqual( round(stats['Tumor', 'LabelmapSegmentStatisticsPlugin.volume_mm3']), 19498.0)
```

Comment:
We might need more tolerance than 0.5mm3. Have a look at the dashboard tomorrow to make sure this test passes on all platforms.

Reply:
@lassoan Seems like the SelfTests are not running on cdash.